LLK/scratch-www
# Fix gh-959: Localize Credits Page

Should at least significantly fix #959 
I did not change any of the strings for translation purposes, but I did make one text change for grammar. (As explained in the test cases)

Test cases:
-

- Check that the page loads English correctly (translated strings aren't likely)
- Check that all the links work
> National Science Foundation, Scratch Foundation, Google, LEGO Foundation, Intel, Cartoon Network, Lemann Foundation, MacArthur Foundation.

should now read 
> National Science Foundation, Scratch Foundation, Google, LEGO Foundation, Intel, Cartoon Network, Lemann Foundation, and the MacArthur Foundation.

I'm hoping there aren't any little typos/gotchas hiding from me, but they could still be there.
